.github/ISSUE_TEMPLATE/Meeting.md

```diff
@@ -51,4 +51,4 @@ https://finos.webex.com/finos/j.php?MTID=m4b1d85127d1f3ca179545d6bd3291975
 
 **Project Wiki Page:** https://finosfoundation.atlassian.net/wiki/spaces/FDX/pages/904626436/Cloud+Service+Certification+Project 
 
-**Mailing List:** Email fdx-cloud-service-certification+subscribe@finos.org to subscribe to our mailing list
\ No newline at end of file
+**Mailing List:** Email fdx-cloud-service-certification+subscribe@finos.org to subscribe to our mailing list
```

Comment:
Are we keeping this email given the latest changes?

Reply:
Hi @abdullahgarcia - That's a great question. I'll ask @maoo how easy this is to change in Google Groups 👍🏻

Reply:
Hi @abdullahgarcia - The previous CSC email addresses have been updated in `README.md` and `.github/ISSUE_TEMPLATE/Meeting.md` in the commits below ...

- 5253024
- f6018f0

Many thanks,

James 🚀